dask/dataframe/io/parquet.py

```diff
@@ -700,9 +700,8 @@ def _read_pyarrow(fs, fs_token, paths, columns=None, filters=None,
     # Find non-empty pieces
     non_empty_pieces = []
     # Determine valid pieces
-    _open = lambda fn: pq.ParquetFile(fs.open(fn, mode='rb'))
     for piece in dataset.pieces:
-        pf = piece.get_metadata(_open)
+        pf = piece.get_metadata()
```

Comment:
Does `piece` maintain a reference to the filesystem on which the original filesystem was opened?

Reply:
Looks like it [doesn't](https://github.com/apache/arrow/blob/master/python/pyarrow/parquet.py#L811)

Reply:
I suppose the `self.open_file_func` is enough